reactor-core/src/test/java/reactor/core/publisher/HooksTest.java

```diff
@@ -949,13 +949,13 @@ public void eachOperatorLiftsParallelFlux() {
 		AtomicInteger liftCounter = new AtomicInteger();
 		Hooks.onEachOperator(Operators.lift((sc, sub) -> liftSubscriber(sc, sub, liftCounter)));
 
-		StepVerifier.create(ParallelFlux.from(Mono.just(1), Mono.just(1)) //internally converts each rail to a Flux using Flux.from
+		StepVerifier.create(ParallelFlux.from(Mono.just(1), Mono.just(1))
 		                                .log()
 		                                .log())
-		            .expectNext(601, 601) //6x lifts => just,Flux.from,ParallelFlux.from,log,log, back to sequential (shared)
+		            .expectNext(501, 501) //5x lifts => just,ParallelFlux.from,log,log,back to sequential (shared)
```

Comment:
same question

Reply:
same answer... no unnecessary Mono wrapping

Reply:
unnecessary may be from assembly standpoint but that was done in order to observe all the applied operators. Changing that now means breaking behaviour which is not a good way to go

Reply:
I understand that this could be redundancy but we may have unexpected reports from those who relied on that via instrumentation libraries